File: firefly/src/main/java/com/firefly/server/http2/router/impl/RegexPathMatcher.java
package com.firefly.server.http2.router.impl;

/**
 * @author Pengtao Qiu
 */
public class RegexPathMatcher extends AbstractRegexMatcher {
    
    @Override
    public MatchType getMatchType() {
        return MatchType.PATH;
    }

}

File: firefly-net/src/main/kotlin/com/fireflysource/net/http/server/impl/matcher/RegexPathMatcher.kt
package com.fireflysource.net.http.server.impl.matcher

import com.fireflysource.net.http.server.Matcher

class RegexPathMatcher : AbstractRegexMatcher() {

    override fun getMatchType(): Matcher.MatchType {
        return Matcher.MatchType.PATH
    }
}